Author: Milan P. StaniÄ‡ <mps@arvanta.net>
Date:   Thu Jan 143 21:33:42 2022 +0000

musl libc doesn't have c_ospeed and c_ispeed types so disable
build of kwboot with this patch

--- a/tools/Makefile	2022-01-10 18:46:34.000000000 +0000
+++ b/tools/Makefile	2022-01-14 20:26:13.814939412 +0000
@@ -217,8 +217,6 @@
 
 HOSTCFLAGS_ubsha1.o := -pedantic
 
-hostprogs-$(CONFIG_ARCH_KIRKWOOD) += kwboot
-hostprogs-$(CONFIG_ARCH_MVEBU) += kwboot
 hostprogs-y += proftool
 hostprogs-$(CONFIG_STATIC_RELA) += relocate-rela
 hostprogs-$(CONFIG_RISCV) += prelink-riscv
